diff --git a/cache/.placeholder b/cache/.placeholder
deleted file mode 100644
index e69de29..0000000
diff --git a/src/Imdb/Config.php b/src/Imdb/Config.php
index a18b4b2..3946bf9 100644
--- a/src/Imdb/Config.php
+++ b/src/Imdb/Config.php
@@ -127,6 +127,8 @@ class Config
 	 */
 	public $ip_address = '';
 
+	public $bind_ip_address = '';
+
 	/**
 	 * Set hostname of HTTP-Proxy
 	 * @var string
diff --git a/src/Imdb/Request.php b/src/Imdb/Request.php
index 62ab80a..cfcbd1c 100644
--- a/src/Imdb/Request.php
+++ b/src/Imdb/Request.php
@@ -66,6 +66,9 @@ class Request
 		if ($config->ip_address) {
 			$this->addHeaderLine('X-Forwarded-For', $config->ip_address);
 		}
+		if ($config->bind_ip_address) {
+			curl_setopt($this->ch, CURLOPT_INTERFACE, $config->bind_ip_address);
+		}
 	}
 
 	public function addHeaderLine($name, $value)
